This request solves issue #71 (user story-profile page) in specific tasks #73 #74 #78

Swallow666 opened the pull request:
This request set up a basic profile page #73 , add info to it #74 , and users can access it through navbar by clicking their username #78 . Anyone gives me some suggestions on coding details or something else please. There are lots of difficulties on create profile page. Help Needed!!

Swallow666 committed:
This request solves issue #71 (user story-profile page) in specific task #73 #74 #78

This request set up a basic profile page #73 , add info to it #74 , and users can access it through navbar by clicking their username #78 . Anyone gives me some suggestions on coding details or something else please. There are lots of difficulties on create profile page. Help Needed!!

kevcamel reviewed (commented):
Good joob @Swallow666, your commit message should be a bit more descriptive(8431fb3ce0b4885eb11b634ad286fd501b9a0912), say what its doing as well as referencing the issue/task.

Your pull request message is excellent! This would be a good example for your commit messages.

Ill review the code tomorrow 👍

kevcamel reviewed (commented):
A few points I would like to mention:
- Bootstrap V4 no longer supports panels ( you were probalby looking at documentation for V3
- Try and keep your code clean, tabs and indentation is key
- When using columns, if you specify only ONE col ex: col-md-6 means -> md,lg,xl will be 6 and everything below md will be 12
- You can read further here https://v4-alpha.getbootstrap.com/layout/grid/
- I removed your toppad class and replaced with the default built in pt-4 that bootstrap has built in

kevcamel committed:
Code review for PR #119

Resolved profile page layout
Resolved routes for city, phone number retrieval from databas

Swallow666 commented:
@kcamcam  That's nice!
